### PR TITLE
Issue42 mmd mapping

### PR DESCRIPTION
**Pull request by mortenwh**
**Summary**: Updated elements according to issue discussion

**Related issue**: #42 

**Suggested reviewer(s)**: @ferrighi 

**Reviewer checklist**:

- [ ] The headers of all files contain a reference to the repository license (i.e., "License: This file is part of py-mmd-tools, licensed under the Apache License 2.0 (https://www.apache.org/licenses/LICENSE-2.0)")
- [ ] 100% test coverage of new code - meaning:
    - [ ] The overall test coverage increased or remained the same as before
    - [ ] Every function is accompanied with a test suite
    - [ ] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
    - [ ] Functions with optional arguments have separate tests for all options
- [ ] Examples are supported by doctests
- [ ] All tests are passing
- [ ] All names (e.g., files, classes, functions, variables) are explicit
- [ ] Documentation (as docstrings) is complete and understandable

The checklist is based on the S-ENDA conventions and definition of done (see https://s-enda-documentation.readthedocs.io/en/latest/general_conventions.html). The above points are not necessarily relevant to all contributions. In that case, please add a short explanation to help the reviewer.

### CODE REVIEW

File: py_mmd_tools/mmd_elements.yaml [hunk 1]
```diff
@@ -150,19 +150,23 @@ use_constraint:
...
+    acdd: 
+      - creator_role
+      - contributor_role
   name:
     default: 'unknown'
     separator: ','
-    acdd: creator_name
+    acdd:
+      - creator_name
+      - contributor_name
```
Comment (TAlonglong) [inline py_mmd_tools/mmd_elements.yaml:169]: I think we need contributor_email and contributor_organisation also else I get like this:
```
  <mmd:personnel>
    <mmd:role>Metadata author</mmd:role>
    <mmd:name>TESTTESTTEST</mmd:name>
    <mmd:email>unknown</mmd:email>
    <mmd:organisation>unknown</mmd:organisation>
  </mmd:personnel>
```

Comment (mortenwh) [reply py_mmd_tools/mmd_elements.yaml:169]: added now

File: py_mmd_tools/mmd_elements.yaml [hunk 2]
```diff
@@ -171,12 +171,14 @@ personnel:
     default: 'unknown'
     separator: ','
     acdd: creator_email
+    acdd_ext: contributor_email
   phone:
```
Comment (TAlonglong) [inline py_mmd_tools/mmd_elements.yaml:175]: Perfect. This do what I need. Thanks

File: tests/test_nc_to_mmd.py [hunk 3]
```diff
@@ -200,22 +201,65 @@ def test_personnel_multiple_mixed(self):
...
+        #self.assertEqual(value[0]['name'], 'Trygve')
+        #self.assertEqual(value[1]['name'], 'Trygve')
+        #self.assertEqual(value[2]['name'], 'Morten')
+        #self.assertEqual(value[0]['email'], 'trygve@meti.no')
+        #self.assertEqual(value[1]['email'], 'trygve@meti.no')
+        #self.assertEqual(value[2]['email'], 'unknown')
+        #self.assertEqual(value[0]['role'], 'Investigator')
+        #self.assertEqual(value[1]['role'], 'Technical contact')
+        #self.assertEqual(value[2]['role'], 'Investigator')
+
```
Comment (TAlonglong) [inline tests/test_nc_to_mmd.py:215]: Remove unused code

**Review by TAlonglong — APPROVED** [review]
LGTM with a small comment

**Review by ferrighi — APPROVED** [review]

**Pull request merged**